src/Symfony/Contracts/Cache/CacheInterface.php

```diff
@@ -11,31 +11,46 @@
 
 namespace Symfony\Contracts\Cache;
 
+use Psr\Cache\CacheItemInterface;
 use Psr\Cache\InvalidArgumentException;
 
 /**
- * Gets/Stores items from/to a cache.
+ * Covers most simple to advanced caching needs.
  *
  * On cache misses, a callback is called that should return the missing value.
```

Comment:
This section should be moved next to get method

Reply:
moved